sklearn/metrics/_classification.py

```diff
@@ -1463,6 +1469,7 @@ def fbeta_score(
     >>> fbeta_score(y_true, y_pred_empty,
     ...             average="macro", zero_division=np.nan, beta=0.5)
     0.12...
```

Comment:
Let's insert a new code snippet that explains how to pass this scoring function as a scoring kwarg in `cross_val_score` by wrapping it with `make_scorer`.